server.js

```diff
@@ -69,24 +82,41 @@ app.get('/users/add', (req, res) => {
     firstname, lastname, email, username, password, type,
   };
   const insertUser = 'INSERT INTO users SET ?';
-  // eslint-disable-next-line consistent-return
   con.query(insertUser, body, (err) => {
     if (err) {
       return res.send(err);
     }
-    res.send('successfully added user');
+    return res.send('successfully added user');
```

Comment:
You don't need to add a return statement here and also consider sending an HTTP status code like 200/201 instead of a string literal.